### PR TITLE
Adding azure prow tests for openshift-ansible.

### PR DESCRIPTION
**Pull request by kwoodson**
Adding presubmit and postsubmit tests for openshift-ansible.

Relies on https://github.com/openshift/aos-cd-jobs/pull/1480

### CODE REVIEW

File: cluster/ci/config/prow/config.yaml [hunk 1]
```diff
@@ -1112,6 +1112,18 @@ presubmits:
     branches:
     - release-3.10
 
+  - name: test_pull_request_openshift_ansible_launch_azure
+    agent: jenkins
+    labels:
+      master: ci.openshift.redhat.com
+    always_run: true
+    skip_report: true
```
Comment (kwoodson) [inline cluster/ci/config/prow/config.yaml]: When test flakes are under control we can enable gating by removing this line.

File: cluster/ci/config/prow/config.yaml [hunk 2]
```diff
@@ -1112,6 +1112,18 @@ presubmits:
...
+    context: ci/openshift-jenkins/gcp-upgrade
```
Comment (kargakis) [inline cluster/ci/config/prow/config.yaml]:
```yaml
context: ci/openshift-jenkins/azure
```

File: cluster/ci/config/prow/config.yaml [hunk 3]
```diff
@@ -1112,6 +1112,18 @@ presubmits:
     branches:
     - release-3.10
 
+  - name: test_pull_request_openshift_ansible_launch_azure
```
Comment (kargakis) [inline cluster/ci/config/prow/config.yaml]:
```yaml
- name: test_pull_request_openshift_ansible_extended_conformance_azure
```
https://ci.openshift.redhat.com/jenkins/job/test_pull_request_openshift_ansible_extended_conformance_azure/

Comment (kwoodson) [reply cluster/ci/config/prow/config.yaml]: thanks! fixed.

**Comment by kwoodson** [comment]
@kargakis, updated.

**Comment by kwoodson** [comment]
@kargakis, I'll update for 3.10.

**Pull request merged**